cloudmanager/resource_netapp_cloudmanager_connector_gcp.go

```diff
@@ -228,7 +247,16 @@ func resourceOCCMGCPRead(d *schema.ResourceData, meta interface{}) error {
 	occmDetails.GCPProject = d.Get("project_id").(string)
 	occmDetails.Region = d.Get("zone").(string)
 	occmDetails.SubnetID = d.Get("subnet_id").(string)
-	client.GCPServiceAccountPath = d.Get("service_account_path").(string)
+	service_account_path := d.Get("service_account_path").(string)
+	if service_account_path == "" {
```

Comment:
same comment as above.
This section should be extracted into a function as it is repeated, and more complex now.